src/go/k8s/kuttl-test.yaml

```diff
@@ -6,6 +6,7 @@ kindContainers:
   - gcr.io/vectorized/redpanda-k8s-operator:latest
 testDirs:
   - ./tests/e2e
+kindConfig: ./kind.yaml
```

Comment:
Shouldn't this be in separated commit?

Reply:
I thought the traditional history-ordering rules wouldn't apply to this repo for the time being.

Reply:
I thought the traditional history-ordering rules wouldn't apply to this repo for the time being.